lightning/src/chain/channelmonitor.rs

```diff
@@ -948,6 +952,7 @@ impl<Signer: Sign> Writeable for ChannelMonitorImpl<Signer> {
 
 		self.lockdown_from_offchain.write(writer)?;
 		self.holder_tx_signed.write(writer)?;
+		self.allow_automated_broadcast.write(writer)?;
```

Comment:
At first I thought I didn't need to write and read the new field. My first understanding was that monitor updates are replayed on deserialization. After some hours trying to make the test work and reading through the code, I realized that probably not the case and we actually serialize and deserialize the whole state so I added it here too and it seems to work.

Reply:
Makes sense. I changed this in [bec767b](https://github.com/lightningdevkit/rust-lightning/pull/1593/commits/bec767b5ccb7b3ec8640f8b4eb70fcd636323e16).